### PR TITLE
EDSC-4302: Update the secondary toolbar to include updated designs as well as leaflet controls

### PR DESCRIPTION
**Pull request by eudoroolivares2016**
# Overview

### What is the feature?

Updates the secondary toolbar with the improvements post the design. This includes adding tool-tips on the buttons in the secondary tool-bar header and some styling improvements to the tour. It also fixes the spacing discrepancy between the colormap and the scale in leaflet when a colormap is rendered

### What is the Solution?

Updating the styling for the Tour steps, secondary toolbar, and the colormap

### What areas of the application does this impact?

Secondary toolbar, Leaflet colormap, the tour steps visuals

# Testing

### Reproduction steps

- **Environment for testing:*Any*
- **Collection to test with:*C1601063219-NSIDC_ECS* (For the colormap)

1. Ensure that hovering over the following buttons displays a tool-tip (My Project, Save Project, Login, Tour button
2. Ensure that the tour steps continue to work as expected with the following differences: the final page for the webinar has been updated with an updated image for the video link; ensure that when hovering the button reacts by enlarging the play button. Also ensure that clicking on the image takes you to the webinar on Youtube

### Attachments
Logged out
image

Logged in with a project created
image

Webinar last tour step
image

Even out spacing between colormap and the leaflet scale
image

Tour First page:
image

Please include relevant screenshots or files that would be helpful in reviewing and verifying this change.

# Checklist

- [x] I have added automated tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by dpesall — APPROVED** [review]

File: static/src/js/components/Tour/TourSteps.jsx [hunk 1]
```diff
@@ -103,9 +107,9 @@ const TourSteps = ({
           preferences.
         </p>
         <p className="search-tour__note">
-          If you want to skip the tour for now, it is always available by clicking
+          If you want to skip the tour for now, it is always available by clicking the
           {' '}
-          <strong>Start Tour</strong>
+          <strong>?</strong>
```
Comment (macrouch) [inline static/src/js/components/Tour/TourSteps.jsx:112]: Should this be an icon and not just a character?

Comment (eudoroolivares2016) [reply static/src/js/components/Tour/TourSteps.jsx:112]: Do you mean something like 
image
I think that does look nicer and then its the exact same character

Comment (eudoroolivares2016) [reply static/src/js/components/Tour/TourSteps.jsx]: I updated this and fixed the line spacing between the `at` added screenshot to PR

**Review by bnp26 — APPROVED** [review]

**Comment by eudoroolivares2016** [comment]
I'm going to wait to merge until I get the go ahead from UI/UX after this gets in we'll be turning on the tour feature

**Review by trevorlang — CHANGES_REQUESTED** [review]
Making sure this gets blocked until I have a second to review today

**Review by trevorlang — APPROVED** [review]
Making sure this gets blocked until I have a second to review today

**Pull request merged**